docs/ref-info/glossary-uid.md

```diff
@@ -119,6 +123,11 @@ sidebar_position: 10
 <dd>Each UID2 <a href="/docs/intro#participants">participant</a> using a server-side implementation has an API key (client key) and also a secret value associated with the key, called the client secret (API secret). The client secret is known only to the participant and the UID2 service.</dd>
 <dd>For details, see <a href="/docs/getting-started/gs-credentials">UID2 Credentials</a>.</dd>
 
+<dt class="jump-anchor" id="gl-cstg">Client-side token generate (CSTG)</dt>
```

Comment:
_Generate_ or _Generation_? in Prebid.js Integration Overview page, it mentioned as _Generation_

Reply:
I want KK to weigh in on this one. Prebid overview "Passing the UID2 Token to the Bid Stream" uses both. "generation" flows better in some examples but we need to be consistent. TBD and I've noted your query in the Overview page as a query for KK. Based on the decision I'll make sure they are consistent.

Reply:
Discussed. CSTG and SSTG terms are coming out so this is a non-issue.